README.md

```diff
@@ -258,36 +278,52 @@ e test --runners=main
 
 Example usage:
 
-```
+```sh
 $ uname
 Darwin
+
 $ e show exe
 /Users/username/electron-gn-root/src/out/Testing/Electron.app/Contents/MacOS/Electron
 
 $ uname
 Linux
+
 $ e show exe
 /home/username/electron-gn-root/src/out/Testing/electron
+
 $ e show out
 Testing
+
 $ e show src
 /home/username/electron-gn-root/src/electron
-$ cd `e show src base`
-$ pwd
+
+$ cd `e show src base` && pwd
 /home/username/electron-gn-root/src/base
+
 $ ripgrep --t h TakeHeapSnapshot `e show src`
 ```
 
-### `e export-patches [patch-dir]`
+### `e patches [patch-dir]`
 
 Exports patches to the desired patch folder in Electron source tree.
 
-Valid patch directories include:
+Valid patch directories can include:
 
 * `node`
 * `v8`
 * `boringssl`
 * `chromium`
+* `perfetto`
+* `icu`
```

Comment:
Are these two correct? I don't see them in https://github.com/electron/electron/blob/master/patches/config.json

Reply:
@ckerr i included them as they exist on some older branches (see: https://github.com/electron/electron/pull/21969); our patches can run a range but i'd rather have them here for verbosity's sake so folks know they're a possibility